frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -116,8 +123,10 @@ export const TicketDetailsCard = () => {
 					<CircleButton size="medium" variant="viewer" onClick={goNext}><ChevronRight /></CircleButton>
 				</HeaderButtons>
 			</CardHeader>
-			<FormProvider {...useForm()}>
-				<TicketForm template={template} ticket={ticket} />
+			<FormProvider {...formData}>
+				<Form>
```

Comment:
Why is this ``<Form/>`` needed here?

Reply:
Functionally, it is not required, but its a semantic thing since the `<FormProvider>` only creates a context, but it does not add a `<form>` in the actual DOM